test/parallel/test-cli-nodeopt.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+
+// Test NODEOPT env variable.
+
+const assert = require('assert');
+const exec = require('child_process').execFile;
+
+disallow('--version');
+disallow('-v');
+disallow('--help');
+disallow('-h');
+disallow('--eval');
+disallow('-e');
+disallow('--print');
+disallow('-p');
+disallow('-pe');
+disallow('--check');
+disallow('-c');
+disallow('--interactive');
+disallow('-i');
+disallow('--v8-options');
+disallow('--');
+
+function disallow(opt) {
+  exec(process.execPath, {env: {NODEOPT: opt }}, function(err) {
```

Comment:
Style nit: there is a space between `opt` and `}`, but the other curly braces don't use space. Same on line 42.